src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.PlatformNotSupported.cs

```diff
@@ -4428,6 +4428,8 @@ public new abstract class Arm64 : AdvSimd.Arm64
         public static unsafe ulong GetActiveElementCount(Vector<ulong> mask, Vector<ulong> from) { throw new PlatformNotSupportedException(); }
 
 
+        // Read FFR, returning predicate of succesfully loaded elements
```

Comment:
```suggestion
        // Read FFR, returning predicate of successfully loaded elements
```

Reply:
Oh dear, looks like this is wrong in the Arm Arm:
https://docsmirror.github.io/A64/2023-09/rdffr_p_p_f.html

Reply:
However, it has been fixed in the [latest version of the Arm Arm.](https://developer.arm.com/documentation/ddi0487/latest/)